server/manage.py

```diff
@@ -19,4 +20,5 @@ def main():
 
 
 if __name__ == "__main__":
+    runserver.default_port = "3000"
```

Comment:
На этом порту будет запускать ноджс сервер, раздающий реакт статики
```suggestion
    # runserver.default_port = "3000"
```